docs/docs/userguide/resource_management/RESOURCE_SCALING.md

```diff
@@ -18,8 +18,27 @@ The Helm charts provision one `StatefulSet` by default. The number of replicas w
       ```
 
 !!!note "Initial cluster size"
-    **Jira**, **Confluence**, and **Crowd** all require manual configuration after the first pod is deployed and before scaling up to additional pods, therefore when you deploy the product only one pod (replica) is created. The initial number of pods that should be started at deployment of each product is set in the `replicaCount` variable found in the values.yaml and should always be kept as 1.
-For details on modifying the `cpu` and `memory` requirements of the `StatfuleSet` see section [Vertical Scaling](#vertical-scaling-adding-resources) below. Additional details on the resource requests and limits used by the `StatfulSet` can be found in [Resource requests and limits](REQUESTS_AND_LIMITS.md) page.
+      **Jira**, **Confluence**, and **Crowd** all require manual configuration after the first pod is deployed and before scaling up to additional pods, therefore when you deploy the product only one pod (replica) is created. The initial number of pods that should be started at deployment of each product is set in the `replicaCount` variable found in the values.yaml and should always be kept as 1.
+      For details on modifying the `cpu` and `memory` requirements of the `StatfuleSet` see section [Vertical Scaling](#vertical-scaling-adding-resources) below. Additional details on the resource requests and limits used by the `StatfulSet` can be found in [Resource requests and limits](REQUESTS_AND_LIMITS.md).
+
+### Scaling Jira safely
+At present there are issues relating to index replication with Jira when immediately scaling up by more than 1 pod at a time. See [Jira and horizontal scaling](../../troubleshooting/LIMITATIONS.md#jira-limitations-and-horizontal-scaling).
+
+To overcome these issue you need to take the following approach:
+
+* Using either the `declarative` or `impreative` approach scale the cluster by **1 pod only**
+!!!warning "1 pod as a time!"
+      
+      Ensure you only scale up by 1 pod at a time!
+  
+* Make sure that the new pod is running and has a state of `Running` 
+* Log into the Jira instance as the `admin` user via the service URL
+* Navigate to `System` > `Troubleshooting and support tools`
+* Under the `Instance health` tab make sure that `Cluster Index Replication` has a green tick, for example:   
```

Comment:
@apawelczyk-atlassian can you please review this document and especially this section? We need to provide a reliable way for customers to verify if the index is in a consistent state.

Reply:
I found what the problem with "Cluster Index Replication" health check is: it takes at least two minutes to start showing any problems. It's because that's the maximum replication delay it allows. Can we tell admins to wait that long to make sure the health check hasn't failed?

For completely fresh nodes (ones for which the local home was just created and is empty) we could recommend accessing that pod directly and executing an issue search - if it doesn't return any results, the index is broken and there's no point in waiting for the health check.

Reply:
Do you think we should perform this check, including the Cluster Index Replication that is currently prescribed?

Reply:
On second thought maybe we avoid the Cluster Index Replication check completely:
https://jira.atlassian.com/browse/JRASERVER-72262